content/md/products-super-power/improve-data-quality.md

```diff
@@ -4,65 +4,89 @@ themes: products-super-power
 
 
 ![Image data quality dashboard](../img/data-quality-dashboard.png)
 
-# Follow data quality improvements
-The dashboard allows you to follow your improvements both on **enrichment** and **consistency**.
-You can choose the see the evolution of your data quality for:
+
+# Know your data quality
+
+## Follow data quality improvements
+The dashboard allows you to follow the evolution of your data quality for:
 - your entire catalog
 - a chosen category
 - a chosen family
+- for a given locale + channel selection
+
+### Quality score evolution
+You can see at a glance the current quality score (for your selection of locale+channel+catalog/category/family) and its evolution up to 6 months behind.
+
+![Image data quality evolution charts](../img/data-quality-evolution-chart.png)
 
-![Image data quality evolution charts](../img/data-quality-charts.png)
+### Quality score distribution
+You can also follow the evolution of the distribution of the different quality scores (A,B,C,D and E)
+
+![Image data quality distribution charts](../img/data-quality-distribution-chart.png)
 
 
 ::: tips
-To see the distribution of the grades, simply mouse over the chart bar.
+To see the distribution of the quality score, simply mouse over the chart bar.
 :::
 
 ::: success
-In this view, you can see the evolution of your grades for the past 7 days or 4 weeks.
+In this view, you can see the evolution of your grades for the past 7 days, 4 weeks or 6 months.
 :::
 
+## Measure data quality
 
-# Measure data quality
+### Key indicators
+Key indicators deep-dive into one particular criteria that needs to be fulfilled to reach a perfect quality score.
 
-From the dashboard, you can see at a glance the current grades for your favorite categories and families.
+- **Products with an image**: enables you to know the percentage of products with at least 1 image and also how many products need an image to fulfill the criteria.
+- **Products with a good enrichment ratio**: enables to know the percentage of products with at least 80% of enriched attributes and how many products need to be enriched further.
+- **Products with perfect spelling in text product values**: enables you to know the percentage of products without spelling mistakes in the product data and how many products hold correction suggestions for spell-check.
+- **Products properly labelled**: enables you to know the percentage of attributes and options without spelling mistakes and how many hold correction suggestions for spell-check.
+
+::: success
+By clicking on the number of products that need to be worked on, you are led to a product grid with all products that need to be improved on this specific criteria.
+:::
+
+### Categories and families
+
+From the dashboard, you can see at a glance the current quality score for your favorite categories and families.
 
 ![Image data quality widgets](../img/data-quality-widgets.png)
```

Comment:
I know it's not part of this PR, but the quality of this image is really bad. If you can re-shot it, it'd be great!

Reply:
Yes, Lucas will rework on all screenshots!